MaterialDesignThemes.Wpf/CardAssist.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace MaterialDesignThemes.Wpf;
+
+public static class CardAssist
```

Comment:
I would rename this to `FlipperAssist` since the consumption is designed for flippers not cards, even though the target of the property is a card.

Reply:
I agree, I'll rename it.